server/src/controllers/Events/resolver.ts

```diff
@@ -200,13 +259,36 @@ ${unsubscribe}
     if (!ctx.user) throw Error('User must be logged in to confirm RSVPs');
     const rsvp = await prisma.rsvps.findUnique({
       where: { user_id_event_id: { user_id: userId, event_id: eventId } },
+      include: { events: true },
     });
 
     const rsvpData: Prisma.rsvpsUpdateInput = {
       confirmed_at: new Date(),
       on_waitlist: false,
     };
 
+    const subscribedRoles = await prisma.user_event_roles.findMany({
+      where: {
+        user_id: userId,
+        event_id: eventId,
+        subscribed: true,
+      },
+    });
+
+    if (subscribedRoles) {
```

Comment:
```suggestion
    if (subscribedRoles.length) {
```
Otherwise it will always add a reminder.